crates/node/service/src/actors/traits.rs

```diff
@@ -16,12 +16,21 @@ pub trait ActorContext: Send {
 ///     - Perform background tasks.
 /// - Emit new events for other actors to process.
 #[async_trait]
-pub trait NodeActor {
+pub trait NodeActor: Send + 'static {
     /// The error type for the actor.
     type Error: std::fmt::Debug;
     /// The communication context used by the actor.
+    /// These are the channels that the actor will use to receive messages from other actors.
     type Context: ActorContext;
+    /// The outbound communication channels used by the actor.
+    /// These are the channels that the actor will use to send messages to other actors.
+    type OutboundData: Sized;
```

Comment:
Small nit - `Context` isn't really clear, given that it's now purposed to just be the inbound channels. Maybe we rename it to `InboundData`, and rename `ActorContext` to `CancellableActor` or something?